academic_validation_framework/examples/comprehensive_example.py

```
@@ -0,0 +1,435 @@
+    try:
+        # Run comprehensive validation
+        session = await runner.run_comprehensive_validation(
+            research_output=SAMPLE_RESEARCH_OUTPUT,
+            session_id="basic_validation_example",
+            discipline="Medicine",
+            citation_style="APA",
+            methodology="Systematic Review"
+        )
+        
+        # Print results summary
+        print(f"\n📊 Validation Results Summary:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)